api_schema/schema.go

```diff
@@ -524,6 +524,17 @@ type PostClientPacketCaptureStatsRequest struct {
 	// Report on any errors encounted during apidump that should be shown to the user.
 	ApidumpError     ApidumpErrorType `json:"apidump_error,omitempty"`
 	ApidumpErrorText string           `json:"apidump_error_text,omitempty"`
+
+	// Report CPU and memory usage, if available.
+	AgentUsage *AgentUsage `json:"agent_usage,omitempty"`
+}
+
+type AgentUsage struct {
+	// Akita agent CPU usage, as a percentage of total CPU availability.
+	RelativeCPU float64 `json:"relative_cpu"`
```

Comment:
This does not seem very helpful without context-- how many CPUs are in the container or VM or host?

I suppose the user might know the context already, but it seems confusing to have this be a relative number.